### ISSUE TITLE
[dataio] dataio-pyshovel remove vi keybindings for up/down/left/right (Trac #1574)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1566]
Remove the old vi keybindings:

```text
Move left, down, up, right
	h j k l
```

Only use the arrow keys for this, since that's what all modern terminals have.

Migrated from https://code.icecube.wisc.edu/ticket/1574
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:11:42", 
    "description": "Remove the old vi keybindings:\n{{{\nMove left, down, up, right\n\th j k l\n}}}\n\nOnly use the arrow keys for this, since that's what all modern terminals have.", 
    "reporter": "david.schultz", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067102580394", 
    "component": "combo core", 
    "summary": "[dataio] dataio-pyshovel remove vi keybindings for up/down/left/right", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-03-02T16:58:23", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160302T23:49:11: 
* david.schultz commented:

> In changeset 2442:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2442"
> searching in the shovel. closes #1565. closes #1566
> ```


* david.schultz changed resolution from "" to "fixed"
* david.schultz changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190111T21:51:34: nwhitehorn commented:

> In changeset 169187:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="169187"
> searching in the shovel. closes #1565. closes #1566
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:11:42: 
* olivas changed milestone from "April 2016" to ""
* olivas commented:

> Milestone April 2016 deleted
